src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -123,7 +123,7 @@ private sealed class DribbleStream : Stream
                 for (int i = 0; i < count; i++)
                 {
                     await _wrapped.WriteAsync(buffer, offset + i, 1);
-                    await Task.Yield(); // introduce short delays, enough to send packets individually but so long as to extend test duration significantly
+                    await Task.Delay(3); // introduce short delays, enough to send packets individually but not so long as to extend test duration significantly
```

Comment:
This is going to add appx 10-15ms per byte, at least on Windows. What impact does that have on test running time? Why was this change needed? And where did "3" come from? I'd expect no difference on Windows btw 1 and 3, add it's less than a default timer quantum.

Reply:
It didn't seem to have a measurable effect on running time.  The dribble stream stuff is only used for a few tests.

I made the change speculatively; I can back out if you think it's not useful.  I honestly don't know if it matters at all.  If it does, Yield is probably sufficient here.

Reply:
> It didn't seem to have a measurable effect on running time.

I'm surprised, but if it really doesn't have a measurable impact, it's fine.

Reply:
Since you mention dribble tests... I had two failures with them but they looked like the thing that `LoopbackServer` closed the connection socket under the client (`System.Net.Sockets.SocketException : An existing connection was forcibly closed by the remote host`), one of the issues on #25880.